tests/unit/moduleapi/auth.tcl

```diff
@@ -68,6 +68,22 @@ start_server {tags {"modules"}} {
         assert_equal [r acl whoami] "default"
     }
 
+    test {modules can redact arguments} {
+        r config set slowlog-log-slower-than 0
+        r slowlog reset
+        r auth.redact 1 2 3 4
+        r auth.redact 1 2 3
+        r config set slowlog-log-slower-than 10000
+        set slowlog_resp [r slowlog get]
+
+        # There will be 3 records, slowlog reset and the
+        # two auth redact calls.
+        assert_equal 3 [llength $slowlog_resp]
+        assert_equal {slowlog reset} [lindex [lindex [r slowlog get] 2] 3]
+        assert_equal {(redacted) 1 (redacted) 3 (redacted)} [lindex [lindex [r slowlog get] 1] 3]
```

Comment:
are we sure we wanna allow redacting the command name from slowlog?
it's still gonna appear in the command stats...

Reply:
I guess I thought this was a good idea at the time, but you're right it shows up a lot of places. By that extension the first argument might also be a subcommand, so it will also show up. I think removing the first argument is probably enough.

Reply:
Thinking about this more I want to leave it. A use case I can think of is just redacting all the arguments, but you still want to have the "slow command" registered somewhere.

Reply:
as discussed, i think of this feature as a way to hide sensitive information (like passwords), and not a way to hide commands.
if we want a way to completely hide commands, we should develop a different feature for that.

regarding the sub-command, i don't care.. we can invest another line of code to prevent redacting argv[1] in case this command has a sub-command, or we can leave it, and only forbid redacting argv[1].